pythran/pythonic/numpy/random/standard_gamma.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef PYTHONIC_NUMPY_RANDOM_STANDARD_GAMMA_HPP
+#define PYTHONIC_NUMPY_RANDOM_STANDARD_GAMMA_HPP
+
+#include "pythonic/include/numpy/random/standard_gamma.hpp"
+#include "pythonic/include/numpy/random/generator.hpp"
+
+#include "pythonic/types/ndarray.hpp"
+#include "pythonic/types/NoneType.hpp"
+#include "pythonic/types/tuple.hpp"
+#include "pythonic/utils/functor.hpp"
+#include "pythonic/numpy/random/gamma.hpp"
+
+#include <random>
+#include <algorithm>
+
+PYTHONIC_NS_BEGIN
+namespace numpy
+{
+  namespace random
+  {
+
+    template <class pS>
+    types::ndarray<double, pS> standard_gamma(double s, pS const &shape)
+    {
```

Comment:
There's a special way to define aliases in pythran, have a look to `include/numpy/diagonal.hpp` and `USING_FUNCTOR`